csvkit/utilities/csvjson.py

```diff
@@ -110,39 +144,48 @@ def dump_json(data, newline=False):
                 feature['type'] = 'Feature'
                 properties = OrderedDict()
                 geoid = None
+                geo_type = 'Point'
                 lat = None
                 lon = None
+                coords = None
 
                 for i, c in enumerate(row):
                     if i == lat_column:
+                        if not c:
```

Comment:
I think the issue is that if `c` is `None`, then `float(c)` raises `TypeError` instead of `ValueError`. Can you make a separate PR that fixes the issue of `c` being `None`, by catching `TypeError`? It's separate from the rest of this issue.